sdk/core/azure-core-serializer-avro-jackson/src/test/java/com/azure/core/serializer/avro/jackson/generatedtestsources/HandOfCards.java

```diff
@@ -0,0 +1,238 @@
+/**
```

Comment:
Should the generated classes also have the license header?

Reply:
Since these are generated classes I'm leaning torwards not having a license header.